### DISCUSSION TITLE
Adding new components

### DISCUSSION THREAD

**Discussion by Shubhdeep12** [corvudev/corvu#59] [Q&A] [answered]
are we planning to add more components like Breadcrumbs, Skeletons, Button, etc.

**Comment by GiyoMoon** [comment, marked as answer]
Hey! Right now corvu is focused on maintaining and adding components that are more advanced/specific and don't overlap with [Kobalte](https://kobalte.dev), another great UI library for Solid. The components you mentioned are all already available in Kobalte, so I wouldn't focus on those. In the future, I may consider adding existing components to corvu if I see the benefit or want to experiment for my own pleasure :)

I myself plan to add a carousel primitive next, and possibly a drag and drop + sortable primitive in the future.
I'm open to suggestions for new, suitable primitives for corvu!

**Comment by Shubhdeep12** [comment]
Great!
thanks for the response.
Let me know if any help is needed in future. 
Would love to help
thanks
